website/docs/ide.md

```diff
@@ -36,7 +40,7 @@ suggest it, run the "Developer: Reload window" command from the command palette.
 Scala CLI support needs to be manually activated for specific files. Open an existing `.scala`
 or `.sc` file,
 or create a new one. Then, with that file opened and focused, run the command
-"Start Scala CLI"
+"Metals: Start Scala CLI server"
```

Comment:
_Drive by comment_

I looked but wasn't able to find it. Are the changes that you've made to the fork of Metals and the VS Code extension available anywhere? I'd like to see them and possibly mimic the changes in `nvim-metals` as well so that if users want to play around with this in Neovim they can as well instead of just VS Code.

Reply:
The commands a bit above,
```text
$ curl -fLo metals-scala-cli.vsix https://github.com/alexarchambault/metals-vscode/raw/scala-cli/metals-1.10.7.vsix
$ code --install-extension metals-scala-cli.vsix
```
are supposed to install an extension with that command. It worked on a machine (different from my dev machine) I tried this on, a bit earlier.

Reply:
(It turned out https://github.com/scalameta/metals-languageclient/pull/305 isn't enough, we needed a few more custom stuff in the VSCode client extension.)

Reply:
Sorry if I wasn't clear. I saw the commands, but I'm curios _what_ changes were made in order to make everything work. If possible I'd like to try it out and also in the future add support in `nvim-metals` as well. So I was curious what changes you made, not how to install the VS Code extension.

Reply:
Actually I found it, nevermind https://github.com/alexarchambault/metals-vscode/commit/ff50dd2ad2cf30c8778aba08a71b39691caf8e1a

Reply:
Yeah, it's just that (and the ability to run a server from a full dependency string).